css/style.css

```diff
@@ -8943,3 +9098,23 @@ textarea.form-control {
   100% {
     stroke-dasharray: 89, 200;
     stroke-dashoffset: -136px; } }
+
+/* fixed issue of menu covering anchors */
+:target::before {
+  content: "";
+  display: block;
+  height: 100px; /* fixed header height*/
+  margin: -100px 0 0; /* negative fixed header height */ }
+
+.projects-section .year-heading a:any-link {
+  text-decoration: none; }
+
+.resources-link-icons .icon-external-link {
+  color: #1eaf84;
+  float: right; }
+
+.back-to-all::before {
```

Comment:
.back-to-all makes it easier to create arrows leading from subpages back to the main pages